backend/delivery/serializers.py

```diff
@@ -0,0 +1,31 @@
+from rest_framework import serializers
+from .models import Delivery
+
+
+class DeliverySerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    customerId = serializers.IntegerField()
+    angelId = serializers.IntegerField()
+    orderId = serializers.IntegerField()
+    estimatedTime = serializers.TimeField()
+    customerConfirmation = serializers.BooleanField()
+    angelConfirmation = serializers.BooleanField()
+
+    def create(self, validated_data):
+        """
+        Create and return a new `Snippet` instance, given the validated data.
```

Comment:
Let's delete all comments that is without any meaning. By the way, all comment written in my code appear only because of copying them from internet. I forgot to delete them, so now that is legacy. In fact, we see comments right in the code very seldom because inline comments difficult to maintain(difficult to make them correspond with code all time)